src/SharpGLTF.Core/Schema2/gltf.TextureInfo.cs

```diff
@@ -4,7 +4,7 @@
 namespace SharpGLTF.Schema2
 {
     [System.Diagnostics.DebuggerDisplay("LogicalTexture[{_LogicalTextureIndex}]")]
-    internal partial class TextureInfo
+    public partial class TextureInfo

```

Comment:
Must be public otherwise generated code MeshExtMeshFeatureIDTexture : TextureInfo does not compile (error: 'TextureInfo is inaccessible due to it's protection level')

Reply:
Eventually, these classes will also need constructors, specialised collections and probably some validation.

For example, I think "attribute" is actually an index to an Accessor, so it will probably require the same wiring used in cesium_outline.

Reply:
its little different, if attribute=0 then there must be a custom vertex attribute with name _FEATURE_ID_0, we can check on that

Reply:
I've added a check for the attribute (https://github.com/vpenades/SharpGLTF/pull/199/files#diff-6ea6b6829d26c72a50a197edad04c83f6a8ada627b9bf96affb2aea89f725801R81) , some more checks to go